fuel-vm/src/tests/validation.rs

```diff
@@ -67,6 +67,40 @@ fn transaction_can_be_executed_after_maturity() {
     assert!(result.is_ok());
 }
 
+#[test]
+fn transaction__execution__works_before_expiration() {
+    let arb_max_fee = 1;
+
+    let rng = &mut StdRng::seed_from_u64(2322u64);
+
+    // Given
+    const EXPIRATION: BlockHeight = BlockHeight::new(2);
```

Comment:
```suggestion
    const EXPIRATION: BlockHeight = BlockHeight::new(1);
```

to clearly show that transactions expire only when the current block height is **higher** than the specified expiry block (if the match, the transaction is still ok).

Reply:
For the record, I'd also add a similar test `fn transaction__execution__doesn_not_work_after_expiration()` and use, for example:
```
    const EXPIRATION: BlockHeight = BlockHeight::new(9);
    const BLOCK_HEIGHT: BlockHeight = BlockHeight::new(10);
```